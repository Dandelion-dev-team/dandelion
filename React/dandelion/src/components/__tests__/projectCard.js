import React from "react"
import renderer from "react-test-renderer"
import ProjectCard from "../projectCard"

describe("Project Card", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ProjectCard id={1} image={'testlink'} title={'Test Title'} description={"Test description"}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})