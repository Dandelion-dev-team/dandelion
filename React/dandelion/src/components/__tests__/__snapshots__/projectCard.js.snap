// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Card renders correctly 1`] = `
<div
  className="card"
>
  <div
    className="cardImage"
  >
    <img
      src="testlink"
    />
  </div>
  <div
    className="text"
  >
    <div>
      <h2>
        Test Title
         
      </h2>
    </div>
    <h4>
      Test description
    </h4>
  </div>
  <div
    className="center"
  >
    <div
      class="vl"
    />
  </div>
  <div
    className="experimentList"
  >
    <div
      className="experimentCheck"
    >
      <input
        id="experiment_id"
        name="topping"
        type="checkbox"
        value="experiment_ID"
      />
       Experiment Title
    </div>
    <div
      className="experimentCheck"
    >
      <input
        id="experiment_id"
        name="topping"
        type="checkbox"
        value="experiment_ID"
      />
       Experiment Title
    </div>
    <div
      className="experimentCheck"
    >
      <input
        id="experiment_id"
        name="topping"
        type="checkbox"
        value="experiment_ID"
      />
       Experiment Title
    </div>
    <div
      className="experimentCheck"
    >
      <input
        id="experiment_id"
        name="topping"
        type="checkbox"
        value="experiment_ID"
      />
       Experiment Title
    </div>
    <div
      className="experimentCheck"
    >
      <input
        id="experiment_id"
        name="topping"
        type="checkbox"
        value="experiment_ID"
      />
       Experiment Title
    </div>
  </div>
  <div
    className="center"
  >
    <div
      class="vl"
    />
  </div>
  <div
    className="tagList"
  >
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
    <div
      className="tag"
    >
      <h4>
        Tag
      </h4>
    </div>
  </div>
</div>
`;
